### DISCUSSION TITLE
Does translations in Crowdin count as contribution?

### DISCUSSION THREAD

**Discussion by pavlidisnick** [DevToys-app/DevToys#882] [Q&A] [answered]
Does GitHub and Crowdin integrate? Will the translations count as contribution on GitHub when they'll get approved?

**Comment by vishva-karma** [comment]
@pavlidisnick Please excuse for the delay. As given in the [contributing document](https://github.com/DevToys-app/DevToys/blob/main/CONTRIBUTING.md#use-crowdin-preferred), translations are counted as contributions. Whether the contributions are displayed on your GitHub profile is a separate concept.

**Reply by veler** [reply, marked as answer]
I noticed translation contribution trend to not appear on the github profile
